2_pvc_destination_gen/pvc-destination-gen.yml

```diff
@@ -35,19 +35,38 @@
         - fail:
           when: n.stat.exists == False
```

Comment:
@alaypatel07 This is unrelated to your PR. But while you're at it, can you please change `n.stat.exists == False` to `not n.stat.exists`